### DISCUSSION TITLE
How to get active alerts issued by our local WFO for our coverage area?

### DISCUSSION THREAD

**Discussion by ckolovson** [weather-gov/api#675] [Q&A]
I would like to get all active alerts issued by our local WFO. Not sure how to do this. I see that I can get the active alerts in California using this:
https://api.weather.gov/alerts/active?area=CA
How do I filter the results to the SF Bay Area, e.g. cwa=mtr? It seems that the API only supports location based filter parameters, such as point location, county, or zone. Can someone please provide a simple working example, and one that doesn't return duplicates? It seems odd that such a simple and useful query should be so complicated to implement using this API.

**Comment by ckolovson** [comment]
I figured out how to query based on Counties. For example, to get active alerts for San Mateo and Santa Clara Counties, the query is:
https://api.weather.gov/alerts/active?zone=CAC081,CAC085

**Reply by sullynole** [reply]
It does seem there is not a CWA filter option, but that could be accomplished by listing all of the counties in the zone filter.  There are more details on zone queries here: https://www.weather.gov/media/documentation/docs/NWS_Geolocation.pdf
